source/plugin/offline-userplayer-data.rst

```diff
@@ -0,0 +1,71 @@
+=======================================
+Accessing data of offline Users/Players 
```

Comment:
This isn't title case. And even then, it's a rather long title looking at it in the index.
Perhaps `Offline Player Data` ? Or `Offline User Data Storage` ? Or even just `User Storage Service` ?